src/App.tsx

```diff
@@ -22,6 +25,12 @@ function App() {
         }
       />
       <Route path="/login" element={<LoginPage />} />
+      <Route path="/my" element={<PrivateRoute />}>
+        <Route path="/my" element={<MyPage />} />
+      </Route>
+      <Route path="/order/:productId" element={<PrivateRoute />}>
+        <Route path="/order/:productId" element={<OrderPage />} />
+      </Route>
```

Comment:
요렇게 PrivateRoute 하나로 감싸도 괜찮겠네요 😊

```suggestion
<Route element={<PrivateRoute />}>
        <Route path="/my" element={<MyPage />} />
        <Route path="/order/:productId" element={<OrderPage />} />
      </Route>
```